src/components/Navbar/navbar.css

```diff
@@ -74,11 +98,11 @@
 }
 
 .close-hamburger-menu {
-    transform: translateX(1000%);
-    -webkit-transform: translateX(1000%);
-    -moz-transform: translateX(1000%);
-    -ms-transform: translateX(1000%);
-    -o-transform: translateX(1000%);
+    transform: translateX(-100%);
+    -webkit-transform: translateX(-100%);
+    -moz-transform: translateX(-100%);
+    -ms-transform: translateX(-100%);
+    -o-transform: translateX(-100%);
```

Comment:
Appreciation: Good that you are handling everything for all the browsers :)

Reply:
Thank you. This is because of an extension. But I think this is no longer necessary as it is handled by browsers.